view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -153,7 +164,8 @@ define(
                     'first_choice': 300,
                     'ktc': 300,
                     'scb': 500,
-                    'uob': 500
+                    'uob': 500,
+                    'mbb': 500,
```

Comment:
I think MBB minimum should be 500 for every 6 months instead of 500 per month.

Reply:
I made a few changes to show the installments per preferred terms.